atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -142,6 +152,31 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji connection established")
 
+    def _fetch_full_logs(self):
+        if not self.osbs:
+            namespace = get_build_json().get('metadata', {}).get('namespace', None)
+            osbs_conf = Configuration(conf_file=None, openshift_uri=self.url,
+                                      use_auth=self.use_auth, verify_ssl=self.verify_ssl,
+                                      namespace=namespace)
+            self.osbs = OSBS(osbs_conf, osbs_conf)
+            res = self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY) or {}
+            build_info = res.get('parent-image-koji-build') or {}
+            self.build_id = build_info.get('id')
+
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException:
+            msg = "unable to get build logs from OSBS"
+            self.log.error(msg)
+            logs = [msg]
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            msg = "This OSBS client does not support get_orchestrator_build_logs; no logs available"
+            self.log.error(msg)
+            logs = [msg]
```

Comment:
Maybe just `[]` in that case? For friendlier emails. Don't really mind either way, just raising the question.